rpc/testing.go

```diff
@@ -26,7 +26,7 @@ import (
 
 // NewTestContext returns a rpc.Context for testing.
 func NewTestContext(t *testing.T) *Context {
-	tlsConfig, err := security.LoadTLSConfigFromDir(security.EmbeddedPrefix + "test_certs")
+	tlsConfig, err := security.LoadTLSConfigFromDir("test_certs")
```

Comment:
Instead of leaking this constant "test_certs" directory, why not just have another security method called: `security.LoadTLSConfigForTests`? Then in security:

    func LoadTLSConfigForTests() (*TLSConfig, error) {
      return LoadTLSConfigFromDir("test_certs")
    }